### PR TITLE
LIS load tests

### PR DESCRIPTION
**Pull request by achaikou**
Black-box lis.load tests based on seen files and dlis experience

### CODE REVIEW

**Review by ErlendHaa — APPROVED** [review]
Looks good 👍

File: python/tests/lis/test_loading.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+def test_filehandles_closed_when_load_fails(tmpdir, assert_error):
+    # TODO: add test files for every step that can fail, as in dlis
+
+    tmp = str(tmpdir.join('file'))
+    shutil.copyfile('data/lis/layouts/truncated_15.lis', tmp)
+
+    # file is truncated, but 2 LFs was already processed successfully
+    with lis.load(tmp) as files:
+        assert len(files) == 2
+    # TODO: change for explicit raise here as assert_error is nor reliable
```
[the remark this answers is not in the document]
Comment (achaikou) [reply python/tests/lis/test_loading.py:32]: It would be nicer to have 
```
with pytest.raises
      lis.load(tmp)
```

if we ever introduce errorhandler and throw.
Throwing is the thing that actually is important for this test as it's supposed to delete all the filehandlers. Error-handled situation is to be dealt with in "non-failing" test.
By not reliable I meant that to make it more reliable one needs to use overload assert_message_count (or what it is called now) or create different test case for every failing function, which should come in future.

Basically that's a TODO for an ErrorHandler change, if it ever happens :D 
Can change the note if you want to though :)
